### ISSUE TITLE
Compile fails after running "./configure OMRGLUE=./example/glue"

### ISSUE THREAD

**Issue by alin00** [eclipse/omr#67]
After running "./configure OMRGLUE=./example/glue" as shown in the README, compilation fails:

```
make[2]: Entering directory `/home/angelal/git/omr.github/omr_glue_static_lib'
g++   -I. -I.././example/glue -I../include_core -I../omr/startup -I../gc/include -I../gc/startup -I../gc/base -I../gc/base/standard -I../gc/verbose -I../gc/verbose/handler_standard -I../gc/stats -I../gc/structs  -DUT_DIRECT_TRACE_REGISTRATION -I../include_core -I../nls -DLINUX -D_REENTRANT -D_FILE_OFFSET_BITS=64 -DIPv6_FUNCTION_SUPPORT -DJ9HAMMER -c  -MMD -fno-exceptions -fno-rtti -fno-threadsafe-statics -fPIC -ggdb -m64 -Wreturn-type -Werror -Wall -Wno-non-virtual-dtor -O3 -fno-strict-aliasing   -o .././example/glue/CollectorLanguageInterfaceImpl.o .././example/glue/CollectorLanguageInterfaceImpl.cpp
.././example/glue/CollectorLanguageInterfaceImpl.cpp:37:28: fatal error: omrExampleVM.hpp: No such file or directory
 #include "omrExampleVM.hpp"
                            ^
compilation terminated.
```

To make this work, you need to explicitly add ```--enable-OMR_EXAMPLE``` to the configure command.

From a certain point of view, it makes sense that --enable-OMR_EXAMPLE is required because I am compiling the example glue. But this is not very user-friendly.

One possible resolution is to default OMRGLUE to example/glue and enable_OMR_EXAMPLE to yes, so that no configure options are required for the trivial standalone case.

Another option is to make --enable_OMR_EXAMPLE set OMRGLUE to example/glue, so that the minimal standalone configuration is ```./configure --enable-OMR_EXAMPLE```.

**Comment by charliegracie** [comment]
That last line was likely a copy and paste error.  The beginning of the doc explaining how to build uses the config makefile which is tbe basic way to do it right now.
make -f run_configure.mk SPEC=linux_x86-64 OMRGLUE=./example/glue

**Comment by alin00** [comment]
You can't use run_configure.mk if you want configure to detect your build environment. This used to work.

**Comment by youngar** [comment]
Closed via #227
